CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+* Refactor `SlaveLag` throttler as `ReplicaLag` (https://github.com/Shopify/lhm/pull/144)
```

Comment:
it's more of a rename than refactor, no? Also I think it's nice to include steps for updating the code inside applications, i.e. rename `SlaveLag` as `ReplicaLag` or `slave_lag_throttler` to `replica_lag_throttler`

Reply:
Good point.

Considering what you said in [this comment](https://github.com/Shopify/lhm/pull/144#discussion_r1310187818), I changed the language to refer to these changes as _deprecations_.